teaspoons-cli/teaspoons/auth_helper.py

```diff
@@ -0,0 +1,94 @@
+# auth_helper.py
+
+import jwt
+import os
+import typing as t
+
+from oauth2_cli_auth import OAuth2ClientInfo, OAuthCallbackHttpServer, get_auth_url, open_browser, exchange_code_for_access_token
+from urllib.parse import quote
+
+from config import CliConfig
+
+
+cli_config = CliConfig()  # initialize the config from environment variables
+
+
+def get_auth_url(client_info: OAuth2ClientInfo, redirect_uri: str) -> str:
+    """
+    Note: this is overridden from then oauth2-cli-auth library to be able to specify a custom auth url
```

Comment:
How does this differ from `oauth2_cli_auth.get_auth_url`? Looks like just the addition of `&prompt=login`?

Wonder if you could call the library method and just append the extra `&prompt=login` field..